test/test_ckan_to_bigquery.py

```diff
@@ -240,11 +240,18 @@ def test_search_where_like_statement(self):
         out = client.search_sql(sql)
         assert len(out['result']['records']) == 223287
 
-    def test_search_limit(self):
+    def test_search_limit_under_threshold(self):
         # returns 32000 records,
         # ignoring the 'LIMIT 32100' in query because we specified limit to 32000 
         # in ckan config 'ckan.datastore.search.rows_max'
         # but limit is more than rows_max so rows_max=32000 wins
-        sql = 'SELECT * FROM %s LIMIT 32100' % table_name
+        sql = 'SELECT * FROM %s LIMIT 10' % table_name
+        out = client.search_sql(sql)
+        assert len(out['result']['records']) == 10
+    
+    def test_search_limit_above_threshold(self):
+        sql = 'SELECT * FROM %s LIMIT 1000000' % table_name
         out = client.search_sql(sql)
         assert len(out['result']['records']) == 32000
```

Comment:
There should be *no* records when above the threshold. Everything should be written to google cloud storage.

Reply:
i.e. behaviour is:

* Below threshold i get results as normal in some big array
* Above threshold i get **no** results inline and everything is in cloud storage

Reply:
yes, the test is a little bit outdated, I will update this.

Reply:
new commit with relevant changes https://github.com/datopian/ckanext-datastore-bigquery/pull/2/commits/04867c376f03f6792a6634ded7682c5df1c34a13